src/DCRoom.php

```diff
@@ -158,6 +158,33 @@ public function showForm($ID, array $options = [])
         echo "</td>";
         echo "</tr>";
 
+        echo "<tr class='tab_bg_1'>";
+        echo "<td><label for='dropdown_vis_cell_width$rand'>" . __('Cell width') . "</label></td><td>";
+        Dropdown::showNumber(
+            "vis_cell_width",
+            [
+                'value'  => $this->fields["vis_cell_width"],
+                'min'    => 1,
+                'max'    => 200,
+                'step'   => 1,
+                'rand'   => $rand
+            ]
+        );
+        echo "</td>";
+        echo "<td><label for='dropdown_vis_cell_height$rand'>" . __('cell height') . "</label></td><td>";
+        Dropdown::showNumber(
+            "vis_cell_height",
+            [
+                'value'  => $this->fields["vis_cell_height"],
+                'min'    => 1,
```

Comment:
Under 10px, it is not really usable.

```suggestion
                'min'    => 10,
```